docs/persistence.rst

```diff
@@ -0,0 +1,81 @@
+.. _persistence:
+
+Secure persistence with skops
+=============================
+
+.. warning::
+
+   This feature is very early in development, which means the API is
+   unstable and it is **not secure** at the moment. Therefore, use the same
+   caution as you would for ``pickle``: Don't load from sources that you
+   don't trust. In the future, more security will be added.
```

Comment:
we should also tag this as experimental, the same way we do in sklearn, so that it's clear from users' code that this is an experimental feature.

Reply:
I don't understand how that relates to the documentation. Can we maybe work on this in another PR?

Reply:
Fair enough. Created this: https://github.com/skops-dev/skops/issues/153